gridstatus/tests/test_ercot.py

```diff
@@ -2,11 +2,36 @@
 import pytest
 
 import gridstatus
+from gridstatus import Markets
+
+
+def check_ercot_spp(df, market, location_type):
```

Comment:
Regarding your earlier comment, we can move these into `test_isos.py` once other ISOs get `get_spp`.